File: Week1/Day2/Ninja_Xp/ex02.py
def get_full_name(first_name, last_name):
    name = input("please enter your first name: ")
    name += " " + input("please enter your last name: ")
    return name

def main():
    full_name = get_full_name("", "")
    print(f"full name is: {full_name}")

main()

File: Week4/Day3/Exercises_XP/ex02.js
function displayStudentInfo(objUser){
    //destructuring
    const {first, last} = objUser;
    console.log(`Your full name is ${first} ${last}`);
    //returning the full name
    return `${first} ${last}`;
}

displayStudentInfo({first: 'Elie', last:'Schoppik'});


